garnish/parse.py

```diff
@@ -43,6 +43,8 @@ def parse_top(top_file):
     # read the file as lines and parse data
     with open(top_file, 'r') as f:
         for line in f.readlines():
+            if line.startswith(";"):
+                continue # Ignore comments
```

Comment:
I'm not sure if this is even possible with comments in `.top` files, but instead of ignoring only entire lines starting with `;` I think it's better to strip anything that comes after `;` on each line.
So something like:
```
line = line.split(';')[0]
```